app/views/collection_drafts/forms/fields/_gridded_range_resolution_fields.html.erb

```diff
@@ -0,0 +1,78 @@
+<div class="row sub-fields">
```

Comment:
I think that these field sets can be done in partials that are shared between them.  I think you would only need to pass in the parent for the help text.
X/Y Dim
Units
Min/Max X/Y Dim
Viewing Angle/Scanning Direction

Reply:
We can spin it off as a tech debt ticket, but I suspect it won't get done anytime soon if it isn't done now.

Reply:
I will take a look, but I think it's possible the field html attributes (id, data-level, etc.) might not come out correctly

Reply:
I didn't think it would work, but I was able to move them into partials and it seems to be working.